### PR TITLE
Edits to getting-started.

### PR DESCRIPTION
**Pull request by novemberkilo**
Thanks @christinerose

### CODE REVIEW

**Review by maiste — APPROVED** [review]
Thanks :+1:

**Review by benmandrew — APPROVED** [review]

File: web-ui/view/documentation.ml [hunk 1]
```diff
@@ -22,18 +22,18 @@ let getting_started =
               p
                 [
                   txt
-                    "It uses metadata from the project's opam and dune files \
-                     to work out what to build, and uses caching to make \
-                     builds fast. It uses the information in the opam files in \
-                     the project to automatically test against multiple OCaml \
-                     versions and OS platforms.";
+                    "OCaml-CI uses metadata from the project's opam and dune \
```
Comment (christinerose) [inline web-ui/view/documentation.ml:25]:
```suggestion
                    "OCaml-CI uses metadata from the project's opam and `dune` \
```

**Review by christinerose — APPROVED** [review]
Just the one monospace suggestion for `dune` files. The rest looks great!

File: web-ui/view/documentation.ml (hunk 1 above)
Comment (novemberkilo) [reply web-ui/view/documentation.ml:25]: The backticks do not result in the markup that we are hoping for -- leaving off for now.

Comment (christinerose) [reply web-ui/view/documentation.ml:25]: Bummer. If not, perhaps Dune should be capitalised as a proper noun if we cannot specify it in monospace for `dune`f files. In the Dune documentation, we differentiate the build system (Dune) by the command and file(s) `dune`.

**Pull request merged**